ttyclock.c

```diff
@@ -218,6 +218,8 @@ void
 draw_number(int n, int x, int y)
 {
      int i, sy = y;
+     unsigned int on  = COLOR_PAIR(2) | A_REVERSE;
+     unsigned int off = COLOR_PAIR(0) | A_NORMAL;
```

Comment:
those seem a little ... terse. what is "on"? color? expand the variable.

why isn't this a `#define` too?

Reply:
Ok, I'm going to use more significant names for "on/off", for example "pixel_on" and "pixel_off".
I've not used a #define because the aim is to calculate the value at the beginning of the function and use it immedialy in the inner for cycle. Since the #define is just a text replacement, it does not implement precalculated values.

Reply:
@anarcat would you use different names than "pixel_on" / "pixel_off"? Can you suggest me? For example:

- colored_reverse
- transparent_normal

would be more descriptive?

Reply:
I will explain:

pixel_on is a bitmask made by:
- COLOR_PAIR(2), defined by "init_color" here https://github.com/xorg62/tty-clock/blob/master/ttyclock.c#L540
- A_REVERSE is the attribute that defines reverse characters (reversed space is a big "pixel")

pixel_off is anothter bitmask made by:
- COLOR_PAIR(0) which means "not visible"
- A_NORMAL is "not reverse" attribute

so I use "pixel_on" to draw the "block characters" which are reversed spaces, supported by both xterm and DECterm, and "pixel_off" to draw the "empty space".

is it good to merge it? ;)